doc/nrf/includes/thread_configure_network.txt

```diff
@@ -0,0 +1,10 @@
+Configure the required Thread network parameters with the ``ot channel``, ``ot panid`` and ``ot masterkey`` commands:
+
+.. code-block:: console
+
+   uart:~$ ot channel 11
+   Done
+   uart:~$ ot panid 0xabcd
```

Comment:
We should notice missing setting the panid earlier.
With the default 0xffff the stack will not start.

Reply:
You are right, I saw this problem once but forgot to re-check.